src/screens/Header.js

```diff
@@ -38,6 +44,16 @@ const HeaderPage = () => (
         },
       }}
     />
+
+    <Property name="sticky">
```

Comment:
Need to wrap `Property` with `Properties`
```suggestion
  <Properties>
    <Property name="sticky">
```

Reply:
done